app/model/exactTarget/SubscriptionDataExtensionRow.scala

```diff
@@ -42,21 +52,18 @@ object SubscriptionDataExtensionRow {
         "EmailAddress" -> personalData.email,
         "Subscription term" -> formatSubscriptionTerm(billingPeriod),
         "Payment amount" -> formatPrice(ratePlanCharge.price),
-        "Default payment method" -> formatPaymentMethod(paymentMethod.`type`),
         "First Name" -> personalData.firstName,
         "Last Name" -> personalData.lastName,
         "Address 1" -> address.lineOne,
         "Address 2" -> address.lineTwo,
         "City" -> address.town,
         "Post Code" -> address.postCode,
-        "Default payment method" -> formatPaymentMethod(paymentMethod.`type`),
         //TODO hardcoded!
         "Country" -> "UK",
         "Date of first payment" -> formatDate(subscription.contractAcceptanceDate),
         "Currency" -> formatCurrency(account.currency),
         //TODO to remove, hardcoded in the template
         "Trial period" -> "14",
```

Comment:
The trial period shouldn't be hardcoded anymore. Or will this be part of another PR instead?